### ISSUE TITLE
Evaluate Test dataset

### ISSUE THREAD

**Issue by uthenkam** [rezafuad/ferplus-keras#2]
Hi, Reza Fuad 
Thank you for the way. From training result, I got close to you.
But I found a problem in the validation by use model.evaluate() function . It has very little accuracy.
Can you help me fix or suggest it?

Thank you very much.
Ten

My code:
from keras.optimizers import SGD
input_shape = (64, 64, 1)
num_classes = 8
batch_size = 128
model = VGG13(input_shape, num_classes)

model.load_weights("trained_models/fer2013plus_VGG13_SDG2.67-0.858.hdf5")
sgd = SGD(lr=0.025, decay=0.0005, momentum=0.9, nesterov=True)
model.compile(loss='categorical_crossentropy', optimizer=sgd, metrics=['accuracy'])

score = model.evaluate(X_val, y_val, batch_size=batch_size, verbose=1)
print ("model %s: %.2f%%" % (model.metrics_names[1], score[1]*100))

Result:
25045/25045 [==============================] - 17s 680us/step
model acc: 25.20%

**Comment by rezafuad** [comment]
Hi,

In the testing or evaluation process, you also need to use the preprocessing.image.ImageDataGenerator class.

At least use it with following parameters 
`datagen = ImageDataGenerator(featurewise_center=True, 
                             featurewise_std_normalization=True)
`

Hope it helps.
